.github/workflows/python-package-conda.yml

```diff
@@ -0,0 +1,52 @@
+name: Conda-based pytest
+
+on: [push, pull_request]
+
+jobs:
+  build-linux:
+    runs-on: ubuntu-latest
+
+    strategy:
+      max-parallel: 5
+      matrix:
+        python-version: [2.7, 3.6, 3.9]
+        slycot: ["", "conda"]
+        array-and-matrix: [0]
+        include:
+          - python-version: 3.9
+            slycot: conda
+            array-and-matrix: 1
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python
+      uses: actions/setup-python@v2
+    - name: Set up conda
+      run: |
+        echo $CONDA/bin >> $GITHUB_PATH
+        conda create -q -n test-environment python=${{matrix.python-version}} 
```

Comment:
There is an action for that: https://github.com/marketplace/actions/setup-miniconda

Reply:
... actually I see that you use a github preinstalled miniconda. Missed that. That might be even better than the action.

Reply:
I tried this method on the Slycot workflow, but it does not work with Windows.